app/utils/helpers.rb

```diff
@@ -31,5 +32,7 @@ def open_now?(place, user_now, place_now)
     end
     # Check if 'now' is in range
     user_now >= from && user_now <= to
```

Comment:
if we had these in time objects, you can use Ruby's [Range](https://ruby-doc.org/core-2.6/Range.html)
```ruby
t1 = Time.now
=> 2019-07-15 14:25:18 +0300
t2 = Time.now + 1000
=> 2019-07-15 14:42:05 +0300
t1 .. t2
=> 2019-07-15 14:25:18 +0300..2019-07-15 14:42:05 +0300
(t1..t2).include? Time.now
=> true
(t1..t2).include? Time.now + 2000
=> false
```